front/app/containers/MainHeader/NotificationMenu/components/InternalCommentNotification/index.tsx

```diff
@@ -19,10 +19,16 @@ interface Props {
 
 const mapPostTypeToLink = (
   notification: IInternalCommentNotificationData
-): string => {
+): string | null => {
```

Comment:
TS can deduct this. Now we're saying: this function _should_ return string | null, complain otherwise. It happens to match our code. But best would be to remove it altogether.

Reply:
Explicit types are always faster. It also helps capture the intention when the refactoring this code